tests/integration/session/test_timeout.py

```diff
@@ -11,6 +11,9 @@ def setUp(self):
         settings.EQ_SESSION_TIMEOUT_GRACE_PERIOD_SECONDS = 0
         super().setUp()
 
+    def tearDown(self):
+        settings.EQ_SESSION_TIMEOUT_SECONDS = 3600
```

Comment:
Could you call `tearDown` on the base please? Although the base doesn't implement it at the moment the DynamoDB PR introduces an implementation on it.